Readme.md

```diff
@@ -599,24 +643,79 @@ Ends the last open if or else block.
 
 Defines `Label` with a given value, creating a symbol for it. This can be used similar to `equ`, but symbols can be used before labels are defined and can be used in conjunction with the `.ifdef/.ifndef` conditionals. These can also be useful for declaring symbols for existing code and data when inserting new code.
 
+Unlike `Label:`, note that `.definelabel Label,value` is evaluated only once, thus using any expressions that refer to the current state of the assembler (e.g. `org()`, `.`) in combination with `.definelabel` leads to undefined behavior.
+
+### Function labels
+
+```
+.func     Label
+.function Label
+```
+
+Creates a symbol `Label` with the current memory address as its value. This is equivalent to `Label:` or `.definelabel Label,.`. However, used in conjunction with the `-sym2` command line flag, the size of the function block will also be written to the symfile along with its memory location. A function block must be terminated with `.endfunc/.endfunction`. This is also implicitly invoked when starting another function block.
```

Comment:
As you wrote above, definelabel will not work like that here (and is kind of a misuse in general). Best to omit that.